HousingSearchListener/V1/Infrastructure/QueryResponseExtension.cs

```diff
@@ -0,0 +1,89 @@
+                    var charges = item["consolidated_charges"].L;
+
+                    var chargesItems = charges.Select(p => p.M);
+                    consolidatedChargesList = new List<ConsolidatedChargeDbEntity>();
+                    foreach (Dictionary<string, AttributeValue> inneritem in chargesItems)
+                    {
+                        consolidatedChargesList.Add(new ConsolidatedChargeDbEntity
+                        {
+                            Amount = decimal.Parse(inneritem["amount"].N),
+                            Frequency = inneritem["frequency"].S,
+                            Type = inneritem["type"].S
+                        });
+                    }
+                }
+
+                TenureDbEntity tenure = null;
+                if (item.Keys.Any(p => p == "tenure"))
+                {
+                    var _tenure = item["tenure"].M;
+                    tenure = new TenureDbEntity();
+                    tenure.FullAddress = _tenure["fullAddress"].S;
+                    tenure.TenureId = _tenure["tenureId"].S;
+                    tenure.TenureType = new TenureTypeDbEntity
+                    {
+                        Code = _tenure["tenureType"].M["code"].S,
+                        Description = _tenure["tenureType"].M["description"].S
+                    };
+                    if (_tenure.ContainsKey("primaryTenants"))
+                    {
+                        tenure.PrimaryTenants = new List<PrimaryTenantsDbEntity>();
+                        foreach (var primaryItems in _tenure["primaryTenants"].L)
+                        {
+                            tenure.PrimaryTenants.Add(new PrimaryTenantsDbEntity
+                            {
+                                FullName = primaryItems.M["fullName"].S,
+                                Id = Guid.Parse(primaryItems.M["id"].S)
+                            });
+                        }
+                    }
+                }
+
+                accounts.Add(new AccountDbEntity
+                {
+                    Id = Guid.Parse(item["id"].S),
+                    AccountBalance = decimal.Parse(item["account_balance"].N),
+                    ConsolidatedCharges = consolidatedChargesList,
+                    Tenure = tenure,
+                    TargetType = Enum.Parse<TargetType>(item["target_type"].S),
+                    TargetId = Guid.Parse(item["target_id"].S),
+                    AccountType = Enum.Parse<AccountType>(item["account_type"].S),
+                    RentGroupType = Enum.Parse<RentGroupType>(item["rent_group_type"].S),
+                    AgreementType = item.ContainsKey("agreement_type") ? item["agreement_type"].S : null,
+                    CreatedBy = item.ContainsKey("created_by") ? item["created_by"].S : null,
+                    LastUpdatedBy = item.ContainsKey("last_updated_by") ? item["last_updated_by"].S : null,
+                    CreatedAt = DateTime.Parse(item["created_at"].S),
```

Comment:
Hi Hanna, could you please confirm if all of those fields are needed for the accouns entity - will they all be used for searching accounts / displaying accounts search response?